libopae/plugins/ase/api/src/plugin.c

```diff
@@ -0,0 +1,188 @@
+// Copyright(c) 2018, Intel Corporation
```

Comment:
2018-2019 or just 2019, if it was created this year.

Reply:
will fix